Content.Shared/Chemistry/ReactiveSystem.cs

```diff
@@ -38,6 +38,10 @@ public void ReactionEntity(EntityUid uid, ReactionMethod method, ReagentPrototyp
         if (!TryComp(uid, out ReactiveComponent? reactive))
             return;
 
+        // custom event for bypassing reactivecomponent stuff
+        var ev = new ReactionEntityEvent(method, proto, reagentQuantity, source);
```

Comment:
as far as i understand - thats for bypassing requirement for artifact to have extended components (making it container for solution) which are not really useful. Event by itself, as means to bypass some part of code, seems SUS. Bu overall - we pass solution, so we could even reduce amount of reagent in solution, it we would need to!